### DISCUSSION TITLE
Upgrading locally to Oqtane 3.0

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1809] [Q&A]
On my local environment whilst trying to upgrade getting the connection error : 

`[Oqtane.Infrastructure.DatabaseManager] A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - The certificate chain was issued by an authority that is not trusted.)`

Its SQL Server, running in Docker and IIS.  I have upgraded the Hosting Bundle to .NET 6.
I have updated the ConnectionString with the need attribute Encrypt=false; 

![image](https://user-images.githubusercontent.com/9447612/141774071-a6c5e681-d72e-4baa-9776-0855c3cb99fb.png)

Running a clean install, no problems at all.

Anyone else, and if so, is there a solution?

**Comment by JanOlsmar** [comment]
I had similar error on IIS, changed Aoolication pool identity to LocalSystem and it worked.

**Reply by leigh-pointer** [reply]
Hi @JanOlsmar,
My Application Pool is already set to LocalSystem so unfortunately that is not going to help me, but much appreciated.

**Reply by JanOlsmar** [reply]
It was an odd error and I didn't understood why it worked after the change. So maybe just a change needed was my reaction.  I had tryed to create a new website and put it there.

**Reply by sbwalker** [reply]
@leigh-pointer is your local environment using the multi-tenancy capabilities? If so you will need to update the DBConnectionString field values in the Tenants table in your "master" database ( the one identified in appsettings.json ) to include the Encrypt=false; attrubute as well.

**Reply by sbwalker** [reply]
@leigh-pointer you should also look at the error.log file in the Oqtane.Server\Content\Log folder to see if there are any more error details.

**Reply by leigh-pointer** [reply]
@sbwalker that sorted it, DBConnectionString field values in the Tenants table, thanks.  The error message is a miss leading factor for sure.

**Reply by sbwalker** [reply]
I do understand why Microsoft introduced this breaking change to SqlClient in .NET 6... however it is definitely going to cause a LOT of frustration for applications and installations which relied upon the legacy behavior.

**Reply by JanOlsmar** [reply]
Do the Master value in  tenant table change if you change appsetting.json?  Or do I need to do that manually if I move the app to another server. I guess i have to do it manually.

**Reply by leigh-pointer** [reply]
@JanOlsmar no is doesn't change in tenant table.  This is something currently done manually.  I feel a little foolish that I forgot about this.
